frontend/public/index.html

```diff
@@ -8,6 +8,7 @@
     <link rel="shortcut icon" href="%PUBLIC_URL%/public/favicon.png">
     <title>Distense</title>
     <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" />
+    <link href="//cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.12/semantic.min.css" rel="stylesheet" />
```

Comment:
If we have `semantic-ui` in both `node_modules` and in index.html -- that just speeds it up or why do that?

Reply:
You can use a loader to include the css, but this was just easier -- and if we serve from the cdn its possible that someone could have it browser cached when hitting our page.

Reply:
Ah I see -- that makes sense.  Probably lots will!